### ISSUE TITLE
Discrepancies in hyper-parameters between paper and README

### ISSUE THREAD

**Issue by netw0rkf10w** [facebookresearch/mae#130]
Hello,

In the paper the following hyper-parameters are listed for fine-tuning:

<img width="648" alt="mae_hyperparameters_finetuning" src="https://user-images.githubusercontent.com/8569515/201088309-bbbef8bd-4a78-4bd5-9d66-16ae16f1b0ff.png">

However, in your [README](https://github.com/facebookresearch/mae/blob/main/FINETUNE.md), they are as follows:

```bash
python submitit_finetune.py \
    --job_dir ${JOB_DIR} \
    --nodes 4 \
    --batch_size 32 \
    --model vit_base_patch16 \
    --finetune ${PRETRAIN_CHKPT} \
    --epochs 100 \
    --blr 5e-4 --layer_decay 0.65 \
    --weight_decay 0.05 --drop_path 0.1 --reprob 0.25 --mixup 0.8 --cutmix 1.0 \
    --dist_eval --data_path ${IMAGENET_DIR}
```

I can spot the following differences with respect to what is reported in the paper:

- Learning rate: 5e-4 instead of 1e-3
- Layer-wise decay: 0.65 instead of 0.75.

Using the values in the README we can actually reproduce the results. Does this mean that the values listed in the paper are not correct?

In addition, could you please share the hyper-parameters for **training from scratch**? Unfortunately I was unable to reproduce the 82.3% top-1 accuracy for ViT-B using the hyper-parameters for fine-tuning (the ones from the screenshot above).

Thank you very much in advance for your responses!

**Comment by rayleizhu** [comment]
What's your reproduced result for **ViT-B**?

I strictly follow the hyperparameters in PRETRAIN.md and FINETUNE.md with no modifications, the final fituning top-1  ACC I got is 83.35. Is this normal?
